keps/sig-cli/1441-kubectl-debug/README.md

```diff
@@ -300,6 +307,124 @@ Examples:
   kubectl debug node/mynode -it --image=busybox
 ```
 
+### Debugging Profiles
+
+Since launching `kubectl debug` we've received feedback that more configurability
+is needed for generated pods and containers.
+
+* [kubernetes/kubernetes#97103]: ability to set capability `SYS_PTRACE`
+* [kubernetes/kubectl#1051]: ability to set privileged
+* [kubernetes/kubectl#1070]: strip probes on pod copy
+* (various): ability to set `SYS_ADMIN` and `NET_ADMIN` capabilities
+
+These requests are relevant for all debugging journeys. That is, a user may want to
+set `SYS_ADMIN` while debugging a node, a pod by ephemeral container, or a pod by copy.
+`kubectl debug` is intended to guide the user through a debugging scenario, and
+requiring the user to specify a series of flags on the command line is a poor experience.
+
+Instead, we'll introduce "Debugging profiles" which are configurable via a single command
+line flag, `--profile`.  A user may then use, for example, `--profile=netadmin` when
+debugging a node to create a pod with the `NET_ADMIN` capaibility.
+
+The available profiles will be:
```

Comment:
I can't tell if these are ordered/additive... if so, that would be helpful to clarify

Reply:
They're independent. Will update.

Reply:
if they're independent, shouldn't the "probes and labels are stripped from pod copies" bits be echoed for each?